assets/airports/eddf.json

```diff
@@ -1108,7 +1108,7 @@
                 ["DF149", "MTR", "TOBAK", "APROX", "MARUN"]
             ]
         },
-        "MTR5C": {
+        "METRO5C": {
```

Comment:
The SID declaration should be the same as the ICAO name, so this should remain MTR5C. Same applies to Ried and Taunus SIDs as well.